sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,220 @@
+
+            - Verify that the transition to SCRAP mode can be re-tried after failed or interrupted attempts.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_yield_test
+      desc: '''Verify that it is possible to switch TAP interfaces while injecting external clock.
+
+            Verify that it is possible to access TAP interfaces while injecting an external clock.
+            This test case is valid only in test life cycle states.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Enable external clock injection.
+            - Connect to CPU JTAG and perform register read/writes to memory/registers.
+            - Switch to LC TAP by manipulating the strap pins.
+            - Verify that read/writes to lc_ctrl work as expected.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_ast_test_execution
+      desc: '''Verify that the tester is able to load/exec test sequences into/from SRAM.
+
+            An expected pattern during manufacturing involves loading small program sequences into
+            SRAM, and use them to run higher level operations such as configuring OTP values in the
+            OTP hardware partition.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Load program into SRAM to execute OTP word write sequences.
+            - Trigger OTP writes to HW partition (e.g. device ID).
+            - Perform reset by toggling POR pin.
+            - Select CPU JTAG via strap pins.
+            - Verify that OTP write was successful.
+            '''
+      tags: ["fpga", "silicon"]
+      stage: V2
+      tests: []
+    }
+
+    {
+      name: manuf_cp_device_info_flash_wr
+      desc: '''Verify that the tester is able to write to flash info pages.
+
+            During early manufacturing stages, the tester writes information for later use into
+            flash info pages.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Load program into SRAM to execute flash write sequences.
+            - Write 32B to the isolated flash info partition. Verify unable to read back.
+            - Write 64B to the flash secret partition. Verify result.
+
+            In silicon, a separate test program must perform the following additional operations.
+            In pre-silicon, they can be part of the same program.
+
+            - Switch to LC TAP at runtime.
+            - Request transition to DEV or PROD state (test both cases).
+            - Verify isolated flash info partition expected data.
```

Comment:
Maybe we should spell out that this requires a reset cycle after the transition.